evergreen/eg_app/views.py

```diff
@@ -12,6 +18,54 @@
 def index(request):
     return render(request,'index.html')
 
+def getFileType(filePath):
+    return mimetypes.guess_type(filePath)[0]  
```

Comment:
I don't think ignoring the encoding will work for UTF-8 (i.e. emoji support). That should be appended if given

Reply:
the index into [0] for first index is not immediately obvious. Please assign that list index to a named variable so there is ont a magic value.